torch/onnx/symbolic_opset9.py

```diff
@@ -433,48 +434,21 @@ def _trunc_divide(g: jit_utils.GraphContext, self, other):
             and other.type().scalarType() is not None
             and symbolic_helper._is_fp(other)
         ):
-            out = g.op("Cast", out, to_i=_C_onnx.TensorProtoDataType.FLOAT)
+            return g.op("Cast", casted, to_i=_C_onnx.TensorProtoDataType.FLOAT)
         else:
-            out = g.op(
+            return g.op(
                 "Cast",
-                out,
+                casted,
                 to_i=_type_utils.JitScalarType.from_name(scalar_type).onnx_type(),
             )
-    else:
-        out = g.op("Cast", out, to_i=_C_onnx.TensorProtoDataType.FLOAT)
-    return out
-
-
-@_beartype.beartype
-def _floor_divide(g: jit_utils.GraphContext, self, other):
-    if symbolic_helper._is_fp(self) or symbolic_helper._is_fp(other):
-        out = true_divide(g, self, other)
-        return g.op("Floor", out)
-    else:
-        # Integer division does trunction rounding
-        div = g.op("Div", self, other)
-        # Division is negative if: self < 0 != other < 0
-        zero = g.op("Constant", value_t=torch.tensor(0, dtype=torch.int64))
-        negative = g.op(
-            "Xor",
-            symbolic_helper._lt_helper(g, self, zero),
-            symbolic_helper._lt_helper(g, other, zero),
-        )
-
-        # For negative numbers with self % other != 0, subtract 1 to round down instead of up
-        mod = g.op("Sub", self, g.op("Mul", div, other))
-        fixup_mask = g.op("And", negative, g.op("Not", g.op("Equal", mod, zero)))
-
-        one = g.op("Constant", value_t=torch.tensor(1, dtype=torch.int64))
-        fixup = g.op("Mul", fixup_mask, one)
-        return g.op("Sub", div, fixup)
```

Comment:
Not sure how these part can be all gone.

Reply:
Removed because integer division no longer does truncation rounding